chart/values.yaml

```diff
@@ -503,8 +503,8 @@ docker-registry:
 minio:
   enabled: true
   fullnameOverride: minio
-  accessKey: # create random accesskey and secretkey
-  secretKey: # create random accesskey and secretkey
+  accessKey: sadsadasdsd # create random accesskey and secretkey
```

Comment:
> This feels a bit like a step backwards. We've had this setup before and deliberately moved to MySQL as chart dependency. I understand the helm chart for [MySQL is deprecated](https://github.com/helm/charts/tree/master/stable/mysql). If we don't find a suitable replacement (maybe [this](https://artifacthub.io/packages/helm/bitnami/mysql)) and must maintain it ourselves, I'd propose we do that as a dedicated child chart, and don't mash it into the main one.
> 
> Also, if we maintain our own chart, let's [use a StatefulSet instead of a Deployment and PVC](https://cloud.google.com/blog/products/databases/to-run-or-not-to-run-a-database-on-kubernetes-what-to-consider).

I absolutely disagree. Trying to stay on version 5.7 with MySQL is the main problem. The bitnami chart is already on version 8.0. Using an older version does not work as the Kubernetes cluster we use is not supporting them. Adding a child chart would produce much overhead and is therefore much more difficult to maintain. This is also not "mashed" in as I have added it besides the existing `db-deployment` and added an additional template. All other values are reused minimizing the complexity.
As this database is meant to be for testing purposes, using a statfulset is not necessary and we lose the benefit of a clean installation in the werft deployments.